src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/MergeDataToSearchResultOperation.java

```diff
@@ -38,19 +40,24 @@ public SearchResultView execute(final CaseType caseType, final List<CaseDetails>
              ))
             .collect(Collectors.toList());
 
-        final List<SearchResultViewItem> viewItems = caseDetails.stream()
-            .map(caseData -> new SearchResultViewItem(caseData.getReference().toString(), caseData.getCaseDataAndMetadata()))
-            .collect(Collectors.toList());
+        final List<SearchResultViewItem>
+            viewItems =
+            caseDetails.stream()
+                       .map(caseData -> new SearchResultViewItem(caseData.getReference().toString(),
+                                                                 caseData.getCaseDataAndMetadata(),
+                                                                 caseType.getCaseFields()
```

Comment:
can we simplify this statement? feels too complex to me. One way would be to move 
```
caseType.getCaseFields() .stream()
                                                                         .filter(caseField -> LABEL_FIELD_TYPE.equals(
                                                                             caseField.getFieldType().getType()))
                                                                         .collect(Collectors.toList())))

                                                                        
to a method in CaseType called `getFieldsByType(String type)`